pages/common/[.md

```diff
@@ -2,24 +2,24 @@
 
 > Check file types and compare values.
 > Returns 0 if the condition evaluates to true, 1 if it evaluates to false.
-> More information: <https://www.gnu.org/software/coreutils/test>.
+> More information: <https://www.gnu.org/software/bash/manual/bash.html#index-test>.
 
-- Test if a given variable is equal to a given string:
+- Test if a given variable is equal/not equal to the specified string:
 
-`[ "{{$VARIABLE}}" == "{{/bin/zsh}}" ]`
+`[ "${{variable}}" {{==|!=}} "{{string}}" ]`
 
-- Test if a given variable is empty:
+- Test if a given variable is [eq]ual/[n]ot [e]qual/[g]reater [t]han/[l]ess [t]han/[g]reater than or [e]qual/[l]ess than or [e]qual to the specified number:
 
-`[ -z "{{$GIT_BRANCH}}" ]`
```

Comment:
This example is useful. I'm not sure I agree with removing it.

Reply:
I can recover it as a separate code snippet. But if I were you I would place this snippet in [Jenkins](https://wiki.jenkins.io/display/JENKINS/Building+a+software+project#Buildingasoftwareproject-belowJenkinsSetEnvironmentVariables) tldr page.